md/languages.md

```diff
@@ -1,70 +1,62 @@
 # Languages
 
-## Languages available by default 
+## Languages available by default
+
+The following languages are officially supported:
 
 * English (US)
 * French
 * Spanish (Latin American)
-* Japanese (from 7.1.3)
+* Japanese
+
+Officially supported language means that:
+* Translation is performed by Bonitasoft.
+* Following Bonita BPM components are translated: Studio, UI Designer and Portal.
 
-## How to change the Bonita BPM Portal user interface language
+## Select BPM Portal user interface language
```

Comment:
For me change is confusing because it imply replace something existing with something else. But I'm fine with any of the two wording.